tests/foreman/ui/test_location.py

```diff
@@ -276,53 +242,53 @@ def test_add_domain_1(self, domain_name):
+        strategy, value = common_locators['entity_deselect']
+        loc_name = gen_string('alpha')
+        domain = entities.Domain(name=domain_name).create()
+        self.assertEqual(domain.name, domain_name)
         with Session(self.browser) as session:
             make_loc(session, name=loc_name)
             self.assertIsNotNone(self.location.search(loc_name))
             self.location.update(loc_name, new_domains=[domain_name])
             self.location.search(loc_name).click()
-            session.nav.wait_until_element(
-                tab_locators["context.tab_domains"]).click()
-            element = session.nav.wait_until_element((strategy,
-                                                      value % domain_name))
+            session.nav.click(tab_locators['context.tab_domains'])
+            element = session.nav.wait_until_element(
+                (strategy, value % domain_name))
             self.assertIsNotNone(element)
 
-    @data(gen_string('alpha', 8),
-          gen_string('numeric', 8),
-          gen_string('alphanumeric', 8),
-          gen_string('utf8', 8),
-          gen_string('latin1', 8))
-    def test_add_user_1(self, user_name):
-        """@test: Create user then add user
-        by using the location name
+    @data(
+        gen_string('alpha'),
+        gen_string('numeric'),
+        gen_string('alphanumeric'),
+        gen_string('utf8'),
+        gen_string('latin1')
+    )
+    def test_add_user(self, user_name):
+        """@test: Create user then add that user by using the location name
 
         @feature: Locations
 
         @assert: User is added to location
 
         """
-        strategy, value = common_locators["entity_deselect"]
-        loc_name = gen_string("alpha", 8)
-        password = gen_string("alpha", 8)
+        strategy, value = common_locators['entity_deselect']
+        loc_name = gen_string('alpha')
+        password = gen_string('alpha')
         user = entities.User(
             login=user_name,
             firstname=user_name,
             lastname=user_name,
-            password=password).create_json()
-        self.assertEqual(user['login'], user_name)
+            password=password,
+        ).create()
+        self.assertEqual(user.login, user_name)
         with Session(self.browser) as session:
             make_loc(session, name=loc_name)
             self.assertIsNotNone(self.location.search(loc_name))
             self.location.update(loc_name, new_users=[user_name])
             self.location.search(loc_name).click()
-            session.nav.wait_until_element(
-                tab_locators["context.tab_users"]).click()
-            element = session.nav.wait_until_element((strategy,
-                                                      value % user_name))
+            session.nav.click(tab_locators['context.tab_users'])
+            element = session.nav.wait_until_element(
+                (strategy, value % user_name))
```

Comment:
nitpick: I __guess__ this should be the correct formatting

```python
element = session.nav.wait_until_element(
    (strategy, value % user_name)
)
```
...ignore, if you're sure I'm not right